bounds.py

```diff
@@ -94,6 +97,57 @@ def estimate_dense(X, n_iter=5, name_func="ours", return_time=False):
     return sigma
 
 
+###############################################################################
+# Delattre2024
+###############################################################################
+
+
+def compute_delattre2024(X, n=None, n_iter=4, return_time=True):
+    """Estimate spectral norm of convolutional layer with Delattre2024.
+
+    From a convolutional filter, this function estimates the spectral norm of
+    the convolutional layer with zero padding using Delattre2024.
+
+    Parameters
+    ----------
+    X : ndarray, shape (cout, cint, h, w)
+        Convolutional filter.
+    n_iter : int, default=4
```

Comment:
Parameter `n` must be documented in all functions.
```suggestion
    n : None | int, default None
        TODO
    n_iter : int, default=4
```